packages/message-kit/src/lib/handlerContext.ts

```
@@ -25,25 +25,32 @@ import {
 } from "@xmtp/content-type-remote-attachment";
 import { ContentTypeReaction } from "@xmtp/content-type-reaction";
 
+export const awaitedHandlers = new Map<
+  string,
+  (text: string) => Promise<boolean | void>
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)